lambdas/generate-map/src/index.js

```diff
@@ -69,8 +69,11 @@ const handler = async (event, context) => {
       source_aws_region: region,
     };
 
+    const { services } = await grpcCloudMapsService
+      .getAwsCrawlerConfigurationPromise(source, { metadata });
+
     const options = AWS_REGION_OVERRIDE ? { region: AWS_REGION_OVERRIDE } : {};
-    const networkCrawler = new AwsCrawler(defaultAwsNetworkMapping, options);
+    const networkCrawler = new AwsCrawler(JSON.parse(services), options);
```

Comment:
Any risk in this throwing a type error; i.e. `JSON.parse({})`?

Reply:
I will take care of such error on UI APP API v6 endpoint side: the payload provided by new API v6 endpoint will be unit tested with AWS Crawler validation. So that we don't lose the check that it is valid JSON with proper structure. So, here, JSON parse error will be purely hypothetical. And it will be handled the same way as any other errors from this Lambda (error are logged to AWS Cloudwatch automatically).

What would be actually nice is the way to collect/access logs or at least be reported by those remote accounts AWS Lambda functions execution errors. For now, we will have to rely on the communication with our customers in case something is wrong during Lambda function execution.